examples/nextjs-zip/app/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+node server.js
```

Comment:
Two updates for this line: 

1. `server.js` is in `standalone` directory. 2. 
2. Add `exec` will allow the nodejs process to receive SIGTERM for graceful shutdown. 

`exec node standalone/server.js`

Reply:
`server.js` is at the root of the artifacts directory, so I updated to `exec node server.js`

```bash
$ pwd
aws-lambda-web-adapter/examples/nextjs-zip/.aws-sam/build/NextjsFunction
$ tree -I node_modules/
.
├── .next
├── node_modules
├── package.json
├── public
│   ├── favicon.ico
│   └── vercel.svg
├── run.sh
└── server.js
```

Reply:
I built this example. The server.js was under `standalone` folder. If this is not intended, the build command in Makefile should be checked. 

[ec2-user@ip-172-31-42-124 nextjs-zip]$ ls -l .aws-sam/build/NextjsFunction/
total 4
-rw-rw-r-- 1 ec2-user ec2-user 44 Sep  2 15:28 run.sh
drwxrwxr-x 5 ec2-user ec2-user 90 Sep  2 15:28 standalone

Reply:
Hmm, I can't reproduce.  You're running `sam build` from `nextjs-zip` directory?

Reply:
Yes. I run `sam build` from `nextjs-zip` directory on an ec2 instance. The OS is Amazon Linux 2. The shell is bash.